spec/features/user_signs_in_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+feature "Signing in" do
+  given(:user) { Fabricate(:user) }
+  
+  scenario "Signing in with correct credentials" do
+    sign_in(user)
+    expect(page).to have_content user.full_name
+  end
+  
+  scenario "Signing in with incorrect credentials" do
+    visit sign_in_path
+    fill_in "Email Address", with: user.email
+    fill_in "Password", with: user.password + "wrong"
```

Comment:
Prefer string interpolation over concatenation. `"#{user.password}wrong"` would be more ruby idiomatic. But rather than this I would just leave out the wrong word "wrong" <- I assume the test would not pass with this.